addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3289,8 +3288,8 @@ class SnippetsMenu extends Component {
                     displayName: snippetEl.getAttribute("name"),
                     category: category,
                     content: snippetEl.children,
-                    thumbnailSrc: escape(snippetEl.dataset.oeThumbnail),
-                    imagePreview: escape(snippetEl.dataset.oImagePreview),
```

Comment:
Note: I don't see the point of escaping something used in a `t-att`, won't this double escape?